lib/jormungandr/test/data/jormungandr/genesis.yaml

```diff
@@ -3062,3 +3062,14 @@ initial:
   - fund:
     - address: ca1swl53wlqt5dnl63e0gnf8vpazgt6g5mq384dmz72329eh4m8z7e5un8q6lg
       value: 100000000000
+
+  # Funds & certificates for the single stake pool
+  # https://input-output-hk.github.io/jormungandr/advanced/03_starting_genesis_praos_blockchain.html
+  - fund:
+      - address: ca1s5nwh0ur9qj4tg5hx334ht9zvqwn9ddz8rv5acwdl997zmqpswdujuwqu9j
+        value: 100000000000000
+      - address: ca1s5498yesws3eee80nxlxqlqetpdstam9x0hkcs7f2k4386uyulydvmzrtja
+        value: 100000000000000
```

Comment:
Do we need this :thinking: ? Aren't the initial certs sufficient?

Reply:
Our existing addresses are single-addresses which shouldn't be possible to stake with.

I simplified this PR to only add a single account-address with `1` lovelace that delegates to the pool.

I guess we would like to test that we can stake faucet-ada from the wallet in the future, and that we would like to make them into grouped addresses.